js/src/dapps/basiccoin/Transfer/Send/send.js

```diff
@@ -17,7 +17,7 @@
 import BigNumber from 'bignumber.js';
 import React, { Component, PropTypes } from 'react';
 
-import { eip20 } from '../../../../contracts/abi';
+import { eip20 } from 'contracts/abi';
```

Comment:
I think I probably would have preferred a `~/contracts/abi` - as is just makes it very clear which is the local modules. (As opposed to the current which could be an alias or a module)